source/includes/reference/_profile_webhooks.md

```diff
@@ -0,0 +1,207 @@
+# Profile Webhooks
+
+Profile webhook subscription is a mechanism that will allow you to receive notifications to your servers whenever various events happen in relation to different resources created under a specific ***profile***. You can subscribe to receive webhooks about resource created under a specific [***application***](#app-webhooks).
+
+Before proceeding, make sure the endpoint where you intend to receive webhooks satisfies the following requirements:
+
+* Has a valid domain name (IPs are disallowed)
+* Listens to HTTPS requests on port `443`
+* Has a valid HTTPS certificate signed by a trusted Certificate Authority - CA (self-signed or expired certificates are not accepted)
+* Does not include any query arguments in the URL
+
+`https://webhooks.example.com/balance-change` is a valid URL; `http://webhooks.example.com:8080/hook.php?type=balance` is not.
+
+You can have multiple subscriptions per event type though be mindful you will receive duplicate callbacks, one for each subscription.
+
+Find out more about webhook events [here](#webhook-events).
+
+## Create
+
+
+> Example Request:
+
+```shell
+curl -X POST "https://api.transferwise.com/v3/profiles/{profileId}/subscriptions" \
+  -H "Authorization: Bearer <your user token>" \
+  -H "Content-Type: application/json" \
+  -d '{
+       "name": "Webhook Subscription #1",
+       "trigger_on": "transfers#state-change",
+       "delivery": {
+         "version": "2.0.0",
+         "url": "https://your.webhook.url/12345"
+         }
+      }'
+```
+
+> Example Response:
+
+```json
+{
+    "id": "72195556-e5cb-495e-a010-b37a4f2a3043", 
+    "name": "Webhook Subscription #1",
+    "delivery": {
+        "version": "2.0.0",
+        "url": "https://your.webhook.url/12345" 
+    },
+    "trigger_on": "transfers#state-change", 
+    "scope": {
+        "domain": "profile", 
+        "id": "<profile ID>"
+    },
+    "created_by": {
+        "type": "user",
+        "id": "<your user ID>",
+    },
+    "created_at": "2008-09-15T15:53:00+00:00Z",
+}
+```
+
+### Request
+
+**`POST https://api.transferwise.com/v3/profiles/{profileId}/subscriptions`**
+
+`profileId` - ID of the profile you are subscribing to. 
+
+All fields listed below are required for creating a webhook subscription.
+
+Field                     | Description                                                             | Format
+---------                 | -------                                                                 | -----------
+name                      | A custom name for your webhook to ease with identification              | Text
+trigger_on                | `transfers#state-change`, `transfers#active-cases` or `balances#credit` | Text
```

Comment:
As mentioned, let's link to some other shared section that

- describes the events
- details each version and their structure

Not sure that exists already.